encryption_works.md

```diff
@@ -190,33 +190,25 @@ It's still much better practice to use OTR without verifying fingerprints than t
 
 ### Logs
 
-Here is an excerpt from the [chat logs](http://www.wired.com/threatlevel/2011/07/manning-lamo-logs), published by Wired, of a conversation between Bradley Manning and Adrian Lamo, who turned him in to authorities:
+OTR will encrypt the content of your chats, but there's another factor to consider when you're chatting with Off-The Record: logs.
 
-> (1:40:51 PM) bradass87 has not been authenticated yet. You should authenticate this buddy.
-> 
-> (1:40:51 PM) Unverified conversation with bradass87 started.
-> 
-> (1:41:12 PM) bradass87: hi
-> 
-> (1:44:04 PM) bradass87: how are you?
-> 
-> (1:47:01 PM) bradass87: im an army intelligence analyst, deployed to eastern baghdad, pending discharge for "adjustment disorder" in lieu of "gender identity disorder"
-> 
-> (1:56:24 PM) bradass87: im sure you're pretty busy...
-> 
-> (1:58:31 PM) bradass87: if you had unprecedented access to classified networks 14 hours a day 7 days a week for 8+ months, what would you do?
-> 
-> (1:58:31 PM) info@adrianlamo.com : Tired of being tired
-> 
-> (2:17:29 PM) bradass87: ?
-> 
-> (6:07:29 PM) info@adrianlamo.com: What's your MOS?
+Here is an excerpt from the chat logs of a conversation between Chelsea Manning (shown here as bradass87) and Adrian Lamo, who turned him in to authorities. They were [published](http://www.wired.com/threatlevel/2011/07/manning-lamo-logs) by Wired.
+
+> (1:40:51 PM) bradass87 has not been authenticated yet. You should authenticate this buddy. \
+> (1:40:51 PM) Unverified conversation with bradass87 started. \
+>(1:41:12 PM) bradass87: hi \
+>(1:44:04 PM) bradass87: how are you? \
+>(1:47:01 PM) bradass87: im an army intelligence analyst, deployed to eastern baghdad, pending discharge for "adjustment disorder" in lieu of "gender identity disorder" \
+>(1:56:24 PM) bradass87: im sure you're pretty busy... \
+>(1:58:31 PM) bradass87: if you had unprecedented access to classified networks 14 hours a day 7 days a week for 8+ months, what would you do?
```

Comment:
The formatting of this quote is messed up, it's all smooshed together in the output HTML (click the "View" button to see the rendered Markdown, courtesy of Github).

My guess is it's either because of the line continuation characters (`\`) or because you need to add spaces after some of the quote markers (`>`).